k8s/namespaces/rpe/draft-store-service/aat.yaml

```
@@ -21,7 +21,7 @@ spec:
         memoryLimits: "3072Mi"
         serviceAccountName: tests-service-account
         keyVaults:
-          draft-store:
+          draft-store-aat:
             excludeEnvironmentSuffix: true
```

Shouldn't this handle ?
```suggestion
            excludeEnvironmentSuffix: true
```

---

yeah it should be false. Updated accordingly